CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -134,11 +354,111 @@ public int DisplayUpdate() {
         return ret;
     }  
 
+    @Override
+    public int OffscreenRender(List<CVKRenderable> hitTestRenderables) {
+        cvkVisualProcessor.VerifyInRenderThread();
+        
+        CVKAssert(cvkDevice.GetDevice() != null);
```

Comment:
Oh I just remembered, can you please convert all of these you come across to CVKAssertNotNull?

Reply:
sure

Reply:
I did a lot of them but I'm sure some slipped through